nana/modules/devs.py

```diff
@@ -209,10 +259,10 @@ async def dc_id_check(_client, message):
         text = "{}'s assigned datacenter is **DC5**, located in **SIN, Singapore, SG**".format(user)
     else:
         text = "{}'s assigned datacenter is **Unknown**".format(user)
-    await msg(message, text=text)
+    await edrep(message, text=text)
 
 
-@app.on_message(Filters.me & Filters.command("id", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("id", Command))
```

Comment:
Codacy found an issue: [get_id is too complex (19) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)